languages/cpp/ExampleUse.md

```diff
@@ -1,21 +1,26 @@
-# EXAMPLES
+# Examples
 
+## Prerequisites
 
-## PREREQUISITES
+### Bitwarden Libraries
+
+Have the two Bitwarden libraries at the same path:
 
-### BITWARDEN Libraries
-One should have two libraries at the same path:
 - `BitwardeClient`
 - `bitwarden_c`
 
-It should look like `libBitwardeClient.dylib` and `libbitwarden_c.dylib` for the macOS.
+For each OS the library files will be the following:
+
+- macOS: `libBitwardeClient.dylib` and `libbitwarden_c.dylib`
+- Linux: `libBitwardeClient.so` and `libbitwarden_c.so`
+- Windows:  `BitwardeClient.dll` and `bitwarden_c.dll`
```

Comment:
```suggestion
- macOS: `libBitwardenClient.dylib` and `libbitwarden_c.dylib`
- Linux: `libBitwardenClient.so` and `libbitwarden_c.so`
- Windows:  `BitwardenClient.dll` and `bitwarden_c.dll`
```

Reply:
Great catch, updated in https://github.com/bitwarden/sdk/pull/854/commits/eec019b5c8623ee1902f53c7804e131c0b405445